src/api/connection.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-var */
```

Comment:
Not needed?

Reply:
It is needed, because of the var below it. It disables the no-var rule, the rule basically says, there shouldn't be vars, so I disabled it, the reason I mentioned in one of the comments below

Reply:
Why isn't the graphql url a const?

Reply:
For the same reason, it is used in another file and I can't import it there